### PR TITLE
Various fixes for the device-identifier

### PR DESCRIPTION
**Pull request by TB-DevAcc**
Various fixes for the current version v0.0.0 of shbdeviceidentifier active on develop.

### CODE REVIEW

**Comment by TB-DevAcc** [comment]
Notable changes include:
- refactored Database class
- added DataLoader class
- introduced utilities folder with various utility files
- refactored and moved collect traffic functions to utilities
- minor cli formatting and command changes

File: device-identifier/shbdeviceidentifier/utilities/capture_utilities.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+
+# noinspection PyPep8Naming
+def convert_Capture_to_DataFrame(cap: Capture) -> pd.DataFrame:
+    # TODO: implement
+    return pd.DataFrame.from_dict({"Test": [1, 2, 3]})
+
+
+# noinspection PyPep8Naming
+def convert_Capture_to_Line(cap: Capture, measurement: str = "packet") -> List[str]:
```
Comment (jo-jstrm) [inline device-identifier/shbdeviceidentifier/utilities/capture_utilities.py:74]: Why capitalize the words? Is that a naming convention I am unaware of?

File: device-identifier/shbdeviceidentifier/utilities/capture_utilities.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    else:
+        logger.error("Continuously capturing traffic is currently not supported.")
+        # TODO: add continuous capture support
+        # logger.info("Sniffing indefinitely... (Ctrl-C to stop)")
+        # gen = cap.sniff_continuously()
+    return cap
+
+
+# noinspection PyPep8Naming
+def convert_Capture_to_DataFrame(cap: Capture) -> pd.DataFrame:
```
Comment (jo-jstrm) [inline device-identifier/shbdeviceidentifier/utilities/capture_utilities.py:68]: Why capitalize the words? Is that a naming convention I am unaware of?

**Review by jo-jstrm — APPROVED** [review]

File: device-identifier/shbdeviceidentifier/utilities/capture_utilities.py (hunk 2 above)
Comment (TB-DevAcc) [reply device-identifier/shbdeviceidentifier/utilities/capture_utilities.py:68]: I felt like this made it clearer that we're talking about a Capture object, not necessarily an arbitrary packet capture file. That being said, I'd be ok with adhering to the PEP convention here as well. 

In general, naming can and should be overhauled for the project as a whole. There is already a ticket for it in the Development Project.

**Pull request merged**